app/(main)/league/[leagueId]/entry/[entryId]/week/Week.tsx

```diff
@@ -218,17 +219,6 @@ const Week = ({ entry, league, NFLTeams, week }: IWeekProps): JSX.Element => {
         <GlobalSpinner data-testid="global-spinner" />
       ) : (
         <>
-          <nav className="py-6 text-primary hover:no-underline">
```

Comment:
Why was this removed?  What if the user goes to change their pick and then decides not to?  Shouldn't they be able to go back if needed without having to select a team again?

Reply:
ya idk what i was thinking. I added it back